pages/linux/pstoedit.md

```diff
@@ -0,0 +1,12 @@
+# pstoedit
+
+> Convert PDF files into various image formats.
+> More information: <http://www.pstoedit.net/>.
+
+- Convert a PDF page to PNG or JPEG [f]ormat:
```

Comment:
```suggestion
- Convert a PDF page to PNG or JPEG format:
```
The long option doesn't seem to be present in the [manual](http://www.calvina.de/pstoedit/pstoedit.htm)